src/submodules/pathfinding/astar.jl

```diff
@@ -59,7 +59,8 @@ to specify the level of discretisation of the space.
   metric used to approximate the distance between any two points.
 
 Utilization of all features of `AStar` occurs in the
-[3D Mixed-Agent Ecosystem with Pathfinding](@ref) example.
+[3D Mixed-Agent Ecosystem with Pathfinding](https://juliadynamics.github.io/Agents.jl/stable/examples/rabbit_fox_hawk/)
```

Comment:
As this is not an external link, perhaps it is worth `@id` the section in the docs and `@ref` here with the id?

Reply:
Done